.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+
+[*.php]
+indent_style = tab
+indent_size = 8
```

Comment:
This setting affects the indentation width of tabs in GitHub.
https://stackoverflow.com/questions/8833953/how-to-change-tab-size-on-github